finite_difference/src/parallel_mod.f90

```diff
@@ -192,7 +192,9 @@ function go_decompose(domainx, domainy,             &
        ntiley = ndomainy
     end if ! automatic determination of tiling grid
 
```

Comment:
I wasn't sure if you meant just for this file or as a public function. I did the later because that master is 1 instead of 0 has got me a couple of times before, so I will be able to use the `on_master()` from NemoLite without caring about the indexing.

Reply:
That's a good idea - I was thinking just locally but it's nicer the way you've done it. (I can't remember why I decided to make master '1' instead of '0' - apologies for the confusion.)